general_rules/Procedure.tex

```diff
@@ -228,34 +196,6 @@ \subsection{Time limits}
 	\item \textbf{Show must go on:} On special cases, the referee may let the robot continue the test for demonstration purposes, but no additional points will be scored.
 \end{enumerate}
 
-
-
-\subsection{Restart}
```

Comment:
Restarts have been used in Stage 2 tests in the pasts. As these tests have a longer duration than stage 1.

Reply:
I don't have enough experience to say whether or not this rule helps for that.
Another question I had was whether or not a team may use this rule when their robot fails at the door. The rule does not specify how much time a team has to fix their robot.
I think this will make running a test slot more of a hastle while there are also multiple attempts per test slot.

Reply:
I leave this one for you to discuss with other TC members.